internal/kubernetes/status/components/cert_manager_csi_driver_spiffe.go

```diff
@@ -49,7 +49,7 @@ func (c *CertManagerCSIDriverSPIFFEStatus) Match(md *MatchData) (bool, error) {
 				}
 			}
 
-			if strings.Contains(container.Image, "cert-manager-csi-driver-spiffe-approver") {
+			if strings.Contains(container.Image, "cert-manager-csi-driver-spiffe-approver:") {
 				found = true
 				c.namespace = pod.Namespace
 				if strings.Contains(container.Image, ":") {
```

Comment:
With the change on line 52, we are now certain that there will be a ":" in the image string. Why did we change line 52?

Reply:
Else the `cert-manager-csi-driver` also matches
It's dumb, really we should add some labels to those projects so we can come up with a better check, but I think that can be done later

Reply:
See this solution: https://github.com/jetstack/jsctl/pull/86